src/aspire/commands/estimate_ctf.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+import click
+
+from aspire import ctf
+
+logger = logging.getLogger("aspire")
+
+
+@click.command()
+@click.option("--data_folder", default=None, help="Path to mrc or mrcs files")
+@click.option("--pixel_size", default=1, type=float, help="Pixel size in A")
```

Comment:
The general problem is that even though python3 will accept unicode by default just fine, depending on the user machine, characters might cause a crash when it gets printed (happened to us in a log message on one of these HPC machines...).  A crashing help menu would also be cute.

I _think_ that the way I escaped it should print correctly most of the time, and avoid crashing in the corner case. It did here when I changed localization, but I didn't explore every possible way it could fail. If someone reports it we can...

Reply:
If you're worried, we can just write “angstrom”. Let's see how this works.